src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,193 @@
+    display: none;
+    visibility: hidden;
+  }
+
+  .pf-m-display-inline-flex {
+    display: inline-flex;
+  }
+
+  // Align right
+  .pf-m-align-right {
+    margin-left: auto;
+  }
+
+  /* stylelint-disable selector-max-compound-selectors */
+  [class*="pf-m-grid"] [data-label] {
+    .pf-c-check,
+    .pf-c-radio {
+      input {
+        grid-column: 1;
+      }
+    }
+  }
+  /* stylelint-enable */
+}
+
+.pf-c-inline-edit__group {
+  display: flex;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__group--item--MarginRight);
+  }
+
+  // Value group
+  &.pf-m-value-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-value-group--item--MarginRight);
+  }
+
+  // Action group
+  &.pf-m-action-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-action-group--item--MarginRight);
+
+    display: none;
+    margin-top: var(--pf-c-inline-edit__action-group--MarginTop);
+  }
+
+  // Icon group
+  &.pf-m-icon-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-icon-group--item--MarginRight);
+  }
+
+  // Enable action group
+  &.pf-m-enable {
+    display: inline-flex;
+  }
+
+  // Enable action group
+  &.pf-m-confirm {
+    display: none;
+  }
+
+  &.pf-m-inline {
+    --pf-c-inline-edit__action-group--MarginTop: 0;
+  }
+
+  // Input group
+  &.pf-m-input-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-input-group--item--MarginRight);
+
+    .pf-m-action-group {
+      --pf-c-inline-edit__input-group--item--MarginRight: var(--pf-c-inline-edit__group--m-input-group--m-action-group--item--MarginRight);
+      --pf-c-inline-edit__action-group--MarginTop: 0;
+    }
+  }
+
+  > :last-child {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+  }
+}
+
+.pf-c-table__action,
+.pf-c-inline-edit__group.pf-m-input-group {
+  .pf-c-inline-edit__group.pf-m-action-group {
+    --pf-c-inline-edit__group--m-action-group--item--MarginRight: 0;
+    --pf-c-inline-edit__action-group--MarginTop: 0;
+  }
+}
+
+.pf-c-inline-edit__input,
+.pf-c-inline-edit__group.pf-m-input-group,
+.pf-c-inline-edit__action.pf-m-save,
+.pf-c-inline-edit__action.pf-m-cancel {
+  display: none;
+  visibility: hidden;
+}
+
+.pf-c-inline-edit__input,
+.pf-c-inline-edit__group.pf-m-input-group {
+  flex: 1;
+}
+
+.pf-c-inline-edit__group.pf-m-column {
+  --pf-c-inline-edit__group--item--MarginRight: 0;
+
+  flex-direction: column;
+}
+
+.pf-c-inline-edit [class*="group"] {
+  align-items: baseline;
+}
+
+// Active
+.pf-c-inline-edit.pf-m-editable,
+.pf-c-inline-edit .pf-m-editable {
+  .pf-c-inline-edit__value,
+  .pf-c-inline-edit__action.pf-m-edit,
+  .pf-c-inline-edit__group.pf-m-value-group,
+  .pf-c-inline-edit__group.pf-m-action-group > * {
+    display: none;
+    visibility: hidden;
+  }
+
+  .pf-c-inline-edit__input,
+  .pf-c-inline-edit__group.pf-m-input-group,
+  .pf-c-inline-edit__action.pf-m-save,
```

Comment:
if pf-m-save and pf-m-cancel are just being toggled between `hidden/visible` do we even need those modifiers ? could we just toggle the group that they sit in?

Reply:
They may not always be in the group.

Reply:
The buttons may not always be inside the group